sql/src/main/java/io/crate/analyze/expressions/ExpressionAnalyzer.java

```diff
@@ -406,6 +407,31 @@ protected static boolean isQuotedSubscript(QualifiedNameReference node) {
         return SUBSCRIPT_LITERAL_PATTERN.matcher(node.toString()).matches();
     }
 
+    @Nullable @VisibleForTesting
```

Comment:
afaik we've never used that and are more happy with using reflection for testing visibility instead of changing our main code for testing purpose. maybe also discuss with @mfussenegger